packages/mattermost-redux/src/actions/groups.ts

```diff
@@ -435,3 +435,23 @@ export function archiveGroup(groupId: string): ActionFunc {
         return {data};
     };
 }
+
+export function createGroupTeamsAndChannels(userID: string): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        try {
+            await Client4.createGroupTeamsAndChannels(userID);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            return {error};
+        }
+
+        dispatch(
+            {
+                type: GroupTypes.CREATED_GROUP_TEAMS_AND_CHANNELS,
```

Comment:
This dispatch isn't used. Thoughts on whether we should dispatch it at all?

Reply:
I think we can remove it